CHANGELOG

```diff
@@ -1,3 +1,12 @@
+Version 1.0.1 (2022-02-28)
```

Comment:
I'm wondering whether it should be 2.0.0 as it's we do backward incompatible dependencies upgrades?

Reply:
yeah